scripts/exregional_run_prdgen.sh

```diff
@@ -10,9 +10,12 @@
 . $USHdir/source_util_funcs.sh
 for sect in user nco platform workflow nco global verification cpl_aqm_parm \
   constants fixed_files grid_params \
-  task_run_post task_run_prdgen ; do
+  task_run_prdgen ; do
   source_yaml ${GLOBAL_VAR_DEFNS_FP} ${sect}
 done
+
+POST_OUTPUT_DOMAIN_NAME=$(uw config realize --output-format yaml \
```

Comment:
```suggestion
post_output_domain_name=$(uw config realize --output-format yaml \
```
Since this is using `uw config realize`  and based on your docs anything that uses uwtools config realize tool should have a lowercase naming convention.

Reply:
I'm not sure how prdgen handles environment variables deep down, so I decided to keep this naming convention from the original. It's likely hard to debug if I change it here, and out of scope for this PR.

Reply:
It's only an environment variable if it's `export`ed -- otherwise it it's a local shell variable that won't be visible to subshells or other programs:
```
~ $ FOO=foo
~ $ export BAR=bar
~ $ bash
~ $ echo $FOO

~ $ echo $BAR
bar
```
So, if it isn't exported (and I couldn't determine that it is, though I could have missed something), it shouldn't be in misleading all-caps.

But it may well still be out-of-scope for this PR.